test/spec/unit/core/adapterManager_spec.js

```diff
@@ -712,9 +726,34 @@ describe('adapterManager tests', () => {
         })
       });
 
+      describe('setBidderSequence', () => {
+        let orgBidderSequence;
+        beforeEach(() => {
+          orgBidderSequence = config.getConfig('bidderSequence');
+          sinon.spy(utils, 'shuffle');
+        });
+
+        afterEach(() => {
+          config.setConfig({ bidderSequence: orgBidderSequence });
```

Comment:
I added a function to the config module for testing.  you can just call `config.resetConfig()`

Reply:
Done